src/ReactTouchHandler.js

```diff
@@ -19,10 +19,13 @@
 import emptyFunction from 'emptyFunction';
 import requestAnimationFramePolyfill from 'requestAnimationFramePolyfill';
 
-var MOVE_AMPLITUDE = 1.6;
-var DECELERATION_AMPLITUDE = 1.6;
-var DECELERATION_FACTOR = 325;
-var TRACKER_TIMEOUT = 100;
+let DEFAULT_TOUCH_CONFIG = {
```

Comment:
Added new constant 'DECELERATION_THRESHOLD', which controls the autoscroll threshold.